src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -366,6 +372,27 @@ private async Task PublishRunFromPackage(Site functionApp, Stream zipFile)
             }
         }
 
+        private async Task PublishRunFromPackageLocal(Site functionApp, Func<Task<Stream>> zipFileFactory)
+        {
+            // Set app setting
+            functionApp.AzureAppSettings["WEBSITE_RUN_FROM_PACKAGE"] = "1";
+
+            var result = await AzureHelper.UpdateFunctionAppAppSettings(functionApp, AccessToken);
+            
+            if (!result.IsSuccessful)
+            {
+                ColoredConsole
+                    .Error
+                    .WriteLine(ErrorColor("Error updating app settings:"))
+                    .WriteLine(ErrorColor(result.ErrorResult));
```

Comment:
I think you might wanna throw here instead?

Reply:
@ahmelsayed - updated! mind taking another look?